README.md

```diff
@@ -94,6 +94,10 @@ For Windows, you can use the following commands:
 * `gradlew spotlessApply` an check and apply formatting to any plain-text file.</br>
 * `gradlew build`  provides a command line to execute build script.</br>
 
+If you are using Git Bash, you can use the following commands:
+* `./gradlew check` quality checks on your project’s code using Checkstyle and generates reports from these checks.
+* `./gradlew spotlessApply` checks and applies formatting to any plain-text file.
+* `./gradlew build` provides a command line to execute build script.
```

Comment:
I think we can avoid this rewriting by adding _GitBash_ in the section above where we have the same for Mac explained(in line#84 to line#88).
Any ideas @dotrachit ?

Reply:
I am sceptical about it because in the previous paragraphs we are talking about Operating Systems but Git Bash is simply a CLI so I think I'll leave it for others to give their opinions.

@luckyman20 @devansh-299 @Bhavnaharitsa

Reply:
Even in the operating systems we can't run those commands anywhere, to get them executed we must be in a terminal. For windows that is command prompt and for linux and mac is their generic terminal, we can execute them from the terminal in Android Studio itself which is indeed the operating system based. So, overall the commands are executed in a cli, and as git-bash is built on top of linux kernel so that doesn't create any difference either.
Ideas and arguments are welcomed ;-)

Reply:
I think the commands should be segregated based on OS(Windows, Linux and Mac). Those give a generic idea. Also, I think Readme already had those.

Reply:
Can you please specify the final changes?Or is the current change sufficient?

Reply:
I think the changes are not needed as I assume that's the conclusion of the discussion above.
Because providing command for every source control management system is tough. In future some other SCM might come up and we cant be providing command for that as well. So what we can do is, we can generalise according to the OS they are using which in the  SCM as well will be more or less the same. I hope you got it.